draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -1709,15 +1737,15 @@ may fail (abort) if expand\_message fails.
 hash_to_field(msg, count)
 
 Parameters:
-- DST, a domain separation tag (see discussion above).
+- DST, a domain separation tag (see {{domain-separation}}).
 - F, a finite field of characteristic p and order q = p^m.
 - p, the characteristic of F (see immediately above).
 - m, the extension degree of F, m >= 1 (see immediately above).
 - L = ceil((ceil(log2(p)) + k) / 8), where k is the security
   parameter of the suite (e.g., k = 128).
 - expand_message, a function that expands a byte string and
   domain separation tag into a uniformly random byte string
-  (see discussion above).
+  (see {{hashtofield-expand}}).
```

Comment:
I don't think `{{section-name}}` works inside code blocks, does it? Maybe I'm remember an old version of the toolchain...

Reply:
Yeah, checked, at least with my toolchain the links don't get resolved. I'm pretty sure that's why we had previously said "see discussion above" and in that discussion linked to the appropriate section.

Also: don't we have simliar things going on in other code blocks in the document? I thought so...

I guess we could manually link things together if we're sure the document is in its final form, but I'm a little nervous about that. I wonder if the markdown processor supports a way of unquoting...

Reply:
We can just manually wire things up for now. The RFC editor will deal with this for us anyway.

Reply:
(Fixed)

Reply:
They'll double check that the section pointers seem sensible? Or you're saying, we give them the markdown and they fix it? (In that case, do we leave a note in the document for the editor?)

Reply:
Yeah, they double check that the pointers are correct. They are very thorough...